docs/pages/access-controls/login-rules/kubernetes.mdx

```diff
@@ -0,0 +1,265 @@
+    # to lowercase, and any external "logins" trait.
+    logins:
+      - 'strings.lower(external.username)'
+      - 'external.logins'
+
+    # The external "groups" trait will be passed through unchanged, all other
+    # traits will be filtered out.
+    groups:
+      - external.groups
+---
+apiVersion: resources.teleport.dev/v1
+kind: TeleportLoginRule
+metadata:
+  name: example-traits-expression-rule
+  labels:
+    example: "true"
+spec:
+  # This rule has a higher priority value, so it will be evaluated after the
+  # "terraform-test-map-rule".
+  priority: 1
+
+  # traits_expression is an alternative to traits_map, which returns all desired
+  # traits in a single expression.
+  traits_expression: |
+    external.put("groups",
+      choose(
+        option(external.groups.contains("admins"), external.groups.add("app-admins", "db-admins")),
+        option(external.groups.contains("ops"), external.groups.add("k8s-admins")),
+        option(true, external.groups)))
+```
+
+Create the Kubernetes resources:
+
+```code
+$ kubectl apply -f login-rules.yaml
+```
+
+List the created Kubernetes resources:
+
+```code
+$ kubectl get loginrules
+NAME                             AGE
+example-traits-expression-rule   8m8s
+example-traits-map-rule          8m8s
+```
+
+Check that the Login Rules have been created in Teleport:
+
+```code
+$ AUTH_POD=$(kubectl get pods -l app=teleport-cluster -o jsonpath='{.items[0].metadata.name}')
+$ kubectl exec -i $AUTH_POD -c teleport -- tctl get login_rules
+kind: login_rule
+metadata:
+  id: 1680225062340767900
+  labels:
+    example: "true"
+    teleport.dev/origin: kubernetes
+  name: example-traits-expression-rule
+spec:
+  priority: 1
+  traits_expression: |
+    external.put("groups",
+      choose(
+        option(external.groups.contains("admins"),
+external.groups.add("app-admins", "db-admins")),
+        option(external.groups.contains("ops"),
+external.groups.add("k8s-admins")),
+        option(true, external.groups)))
+version: v1
+---
+kind: login_rule
+metadata:
+  id: 1680225067068319000
+  labels:
+    example: "true"
+    teleport.dev/origin: kubernetes
+  name: example-traits-map-rule
+spec:
+  priority: 0
+  traits_map:
+    groups:
+    - external.groups
+    logins:
+    - strings.lower(external.username)
+    - external.logins
+version: v1
+```
+
+Test the Login Rules by sending some example input traits to the standard input
+of the `tctl login_rule test` command and having it load all Login Rules from
+the cluster.
+
+```code
+$ echo '{"groups": ["admins", "ops"], "username": ["Alice"], "logins": ["user", "root"]}' | \
+  kubectl exec -i $AUTH_POD -c teleport -- tctl login_rule test --load-from-cluster
+groups:
+- admins
+- ops
+- app-admins
+- db-admins
+logins:
+- alice
+- user
+- root
+```
+
+## Step 3/3. Edit the Login Rules with `kubectl`
+
+Edit the `example-traits-map-rule` to add an extra login (`example`) by writing
+the following to `login-rule-edit.yaml`:
+
+```
```

Comment:
Consider using `diff` with `-` and `+` to demonstrate what's changed in this version.